packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -318,6 +319,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           reminder.booking.endTime.toISOString() || "",
           reminder.booking.eventType?.title || "",
           timeZone || "",
+          reminder.booking.location || "",
```

Comment:
I don't think this is works for dynamic link-based location.

Reply:
Thanks @anikdhabal yes correct! It was showing `integrations:daily`, or others.

Missed the testing for that, as URLs were not properly generating.

But yes now I have made the change and now it should show the meeting URL.
Although I have tested it, but as meeting URL was not generated it did just show `Cal Video`, or others.
Can you please let me know how to quickly setup Cal Video locally so that I can test this. 🙏

Reply:
@anikdhabal Does this looks good now? :slightly_smiling_face:

Reply:
Nah, needs to fix